frontend/src/core/app/helpers.js

```diff
@@ -190,6 +186,14 @@ define(function(require){
 
         },
 
+        ifImageIsCourseAsset: function(url, block) {
```

Comment:
the name of this is a bit weird. can you change it to something like "fixAssetURL" or something a bit more imperative